### ISSUE TITLE
Issues probably stemming from user error 

### ISSUE THREAD

**Issue by A-wolf** [PepperCode1/Continuity#193]
Hi, i am trying to use the mod with [PB Craft](https://bdcraft.net/downloads/purebdcraft-minecraft/optifine-ctm-transitions/) and i seen from other posts it is supposed to work. Yes i got indium installed as i am using sodium. The mod seems to work in general as connected textures from vanilla tweaks specifically [Conntected iron blocks](https://vanillatweaks.net/share#kJpw0R) and it works fine. I have also tested so the order i put my resource packs works on normal optifine so what could it other be which is causeing the issue and/or what more other info would you need ?

Thank you i apricate your work on the mod and help ^-^

**Comment by A-wolf** [comment]
TLDR; Solved kinda just putting the issue here for future refrence

Okay so some updates, I got some of the textures to work after installing another part of the texture pack [optifine-ctm-transitions](https://bdcraft.net/downloads/purebdcraft-minecraft/optifine-ctm-transitions/) (glass, iron, copper, and gold blocks) but not others (diamond and emerald blocks) so after looking into it the code of why they worked diffrently does it seem like one of the [block].properties files was written diffrently from the other which most likely was the cause of the issue and then copying over the format did it indeed solve the issue
